remote_branch_checker/remote_branch_checker.sh

```diff
@@ -249,6 +249,13 @@ ${gitcmd} merge -q --no-edit FETCH_HEAD
 exitstatus=${PIPESTATUS[0]}
 if [[ ${exitstatus} -ne 0 ]]; then
     echo "Error: The ${branch} branch at ${remote} does not apply clean to ${baseref}" | tee -a ${errorfile}
+
+    mergeconflicts="$( ${gitcmd} diff --name-only --diff-filter=U )"
+    if [[ -n "${mergeconflicts}" ]]; then
+        echo "Error: Merge conflict(s) in:" | tee -a ${errorfile}
+        echo "${mergeconflicts}" | tee -a ${errorfile}
```

Comment:
Unless I'm wrong (we don't have any test covering exactly this), I think that the value of the `${mergeconflicts}` variable must be added to the previous line (the `Error: Merge conflict(s) in:` one).

The reason is that the job that is in charge of reporting the results to the tracker only reports lines beginning by `(Error|Warn)`, so the current patch won't show the list of conflicting files. By adding the list to the previous line (that begins with `Error`) it will be reported.

Ciao :-)

Reply:
If I'm not wrong, the output of `git diff --name-only --diff-filter=U` produces multi-line content if there's more than one file having merge conflicts. So, adding it to the previous line would probably only work in some cases, and we would only see the first conflicting line. Also having tabular output for a file list would be great.

How about:

```bash
echo "${mergeconflicts}" | sed 's/^/Error: /' | tee -a ${errorfile}
```

Reply:
That's ok for me, I just was imagining that it would be easier to print all them in the same line and forget say with `echo "${mergeconflicts}" | xargs` or some other similar trick (`paste`...). But, as said, that's ok to me too.

Reply:
I added a fix-up commit.